lib/util/backend.dart

```diff
@@ -57,8 +57,8 @@ class Backend {
   static Future<String> getRole() async {
     // these are thr routes we are going to forward the user to
     Map<String, String> routes = <String, String>{
-      'Patient': 'main-screen/mainscreen',
-      'Doctor': 'study-nurse-screen/studynursescreen'
+      'Patient': '/main-screen/mainscreen',
+      'Doctor': '/study-nurse-screen/studynursescreen'
     };
```

Comment:
This is my code you're changing here, is there a reason for this? It used to work previously.

Reply:
The route names have been changed according to the directory structure as it can be seen in the code.

Reply:
#23 has changed all routes but was missing the leading slashes.

> This is my code you're changing here, is there a reason for this? It used to work previously.

Does it mean that it doesn't work?

Reply:
It means: why bother changing something that works?

Reply:
I had to resolve an merge conflict in Routes, because #23 has made similar changes. So since the changes were necessary anyway, I fixed that at the same time.